var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -82,18 +84,18 @@ class Hdf5(CMakePackage):
             values=('default', 'v114', 'v112', 'v110', 'v18', 'v16'),
             multi=False)
 
-    depends_on('cmake@3.12:', type='build')
+    depends_on('cmake@3.15:', type='build')
 
-    depends_on('mpi', when='+mpi')
+    if sys.platform != 'win32':
+        depends_on('mpi', when='+mpi')
```

Comment:
(question) to be clear, does this mean that building `hdf5` with MPI support on Windows does not require any install of MPI on the system?

Reply:
Ping - I would have assumed that building `hdf5+mpi` on Windows would require an `MPI` installation.